stock_available/models/product_product.py

```diff
@@ -17,20 +17,13 @@ class ProductProduct(models.Model):
     _inherit = "product.product"
 
     def _compute_available_quantities_dict(self):
-        stock_dict = self._compute_quantities_dict(
-            self._context.get("lot_id"),
-            self._context.get("owner_id"),
-            self._context.get("package_id"),
-            self._context.get("from_date"),
-            self._context.get("to_date"),
-        )
         res = {}
         for product in self:
             res[product.id] = {
-                "immediately_usable_qty": stock_dict[product.id]["virtual_available"],
```

Comment:
I'd say this was done this way for performance. What's the issue you're trying to solve?

Reply:
I actually made a mistake, my Odoo version was outdated. It is not relevant anymore.
Sorry for the noise, I've update the PR